plugins/ftdidmx/FtdiWidget.h

```diff
@@ -89,26 +100,28 @@ class FtdiWidgetInfo {
     std::string m_name;
     std::string m_serial;
     int unsigned m_id;
+    int m_vid;
+    int m_pid;
 };
 
-
 /**
  * An FTDI widget
  */
 class FtdiWidget {
  public:
-    static const int VID = 0x0403;  // FTDI Vendor ID
-    static const int PID = 0x6001;  // FTDI Product ID
-
     /**
      * Construct a new FtdiWidget instance for one widget.
      * @param serial The widget's USB serial number
      * @param name The widget's USB name (description)
      * @param id The ID of the device (used only when FTD2XX is the backend)
+     * @param vid The VendorID of the device, defaults to old value of FtdiWidget::VID
```

Comment:
Can you change this, referring to a value that doesn't exist anymore doesn't seem great.